### PR TITLE
add readme file of ipvs

### PR DESCRIPTION
**Pull request by Lion-Wei**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:
```
None
```

### CODE REVIEW

File: pkg/proxy/ipvs/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Linux kernel.
+
+IPVS runs on a host and acts as a load balancer in front of a cluster of real servers. IPVS can direct requests for TCP
+and UDP-based services to the real servers, and make services of real servers appear as irtual services on a single IP address.
+
+### How to use
+
+##### Load IPVS kernel modules
+
+Currently can't load IPVS kernel modules automatically, so first we should use the following command to load IPVS kernel
```
Comment (brendandburns) [inline pkg/proxy/ipvs/README.md]: "Currently the IPVS kernel module can't be loaded automatically, so ..."

**Comment by brendandburns** [comment]
/ok-to-test

One small comment to clean up.

**Comment by Lion-Wei** [comment]
@brendandburns , thanks!

**Comment by resouer** [comment]
@brendandburns @thockin Will we add this to 1.8 release?

**Pull request merged**